schematics/web/authentication/files/src/features/UserProvider/UserProvider.tsx.template

```diff
@@ -54,6 +54,7 @@ export const UserProvider: FC = ({ children }) => {
       });
 
       await refreshUser();
+      return Hub.remove;
```

Comment:
It is not enough. remove form Hub is a method that takes the channel name and callback as arguments. Additionally, the clean up effect in useEffect should return a cleaning function.

Reply:
My fault for not checking the documentation. Fixed in next commit.